celeborn.advisories.yaml

```diff
@@ -0,0 +1,15 @@
+schema-version: 2.0.2
+
+package:
+  name: celeborn
```

Comment:
The package name seems to be `celeborn-0.5` rather than Celeborn

Reply:
OOf, good catch. Thanks.